source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -168,10 +168,9 @@ - (IBAction)applyText:(id)sender
 - (IBAction)deleteAllRows:(id)sender
 {
     [(ACRChatWindow *)self.chatWindow.dataSource deleteAllRows:self.chatWindow];
-    
+
     // clean the retrieved inputs
-    if ([self appIsBeingTested])
-    {
+    if ([self appIsBeingTested]) {
```

Comment:
>{ [](http://example.com/codeflow?start=33&length=1)

this looks wrong, all other open curly braces are added in a new line, is this only an if/else rule?

Reply:
I believe it's controlled by `BraceWrappingAfterControlStatementStyle AfterControlStatement Wrap control statements (if/for/while/switch/..).`
and applied to other controls statement. If not, please let me know.

It's documented in https://clang.llvm.org/docs/ClangFormatStyleOptions.html.